docs/receipts-ocr.md

```diff
@@ -0,0 +1,214 @@
+reimbursements = pd.read_csv('../data/2016-12-06-reimbursements.xz', low_memory=False)
+reimbursements = reimbursements.query('(year >= 2015)')
+
+# "JOIN" dataframes on document id
+data = texts.merge(reimbursements, on='document_id')
+```
+
+### Raw Cloud Vision API responses
+
+The data sent to the API is an image encoded in base 64 (more info [here](https://cloud.google.com/vision/docs/reference/rest/v1/images/annotate#AnnotateImageRequest))
+and the data returned is a JSON described [here](https://cloud.google.com/vision/docs/reference/rest/v1/images/annotate#annotateimageresponse).
+
+The `2017-02-15-receipts-texts-raw.tar.xz` file found on Serenata de Amor's S3
+bucket contains the raw JSON responses returned by the API grouped by
+`document_id` and page number:
+
+```javascript
+{
+  "1": [
+    // First element of the array is the full text of the receipt
+    {
+      "description": "... <FULL TEXT OF THE RECEIPT> ...",
+      "boundingPoly": {
+        "vertices": [
+          // The rectangle where the API found the whole text
+          { "y": 469,  "x": 866  },
+          { "y": 469,  "x": 1753 },
+          { "y": 1413, "x": 1753 },
+          { "y": 1413, "x": 866  }
+        ]
+      },
+      "locale": "pt-PT"
+    },
+    // What folllows is each word found and the location it was found
+    {
+      "description": "Restaurante",
+      "boundingPoly": {
+        "vertices": [
+          // The rectangle where the API found the word
+          { "y": 469, "x": 1009 },
+          { "y": 478, "x": 1187 },
+          { "y": 518, "x": 1185 },
+          { "y": 509, "x": 1007 }
+        ]
+      }
+    },
+    // ... other words here ...
+  ],
+  "2": // Same info for page 2
+}
+```
+
+Most of the time you'll only use the full text provided by
+`2017-02-15-receipts-texts.xz` but, for example, if you want to build some logic
+around analysing the text of specific regions of the receipt you're able to do
+that with the `x` and `y` coordinates above.
+
+## Challenges with OCRing the Chamber of Deputies receipts
+
+There are many challenges in dealing with OCRed info and more specifically for
+the receipts scanned and provided by the Chamber of Deputies we need to keep the
+following in mind:
+
+- Many receipts are filled in by hand and unfortunately handwriting is even
+  harder to parse than "computer text".
+- Some receipts are not scanned carefully and end up being rotated (sometimes it
+  is even scanned up side down).
+- Some scans have really low quality and / or the receipt gets a bit "faded
+  away" due to the thermal paper many places emit, making it harder for the
+  computer to guess what's the text in it.
+
+In other words, there is room for (non trivial) improvements on this OCR process
+(like some image preprocessing) and there are things we can do ourselves in
+order to increase the chances of finding suspicious reimbursements with the PDFs
+provided.
+
+## How can I OCR other receipts?
```

Comment:
@cuducos @jtemporal Hey guys, in the spirit of getting this out the door for more people to use... What do you guys think about removing this section from the docs and merge just the documentation on the dataset as it is?

This should not only make this data more visible to other contributors but also allow 3 or 4 other PRs I got open with new notebooks to be merged. I'm still interested in making the code I outlined here work, but other people can benefit from this information and try to come up with new insights while I find the time to work on that.